src/components/charts/selectors/chart-selectors.js

```diff
@@ -48,13 +55,7 @@ const getDataMax = createSelector(getDataWithTotal, data => {
 });
 
 export const getDomain = createSelector(
-  [
-    getData,
-    state => state.config,
-    state => state.projectedData,
-    getDataMin,
-    getDataMax
-  ],
+  [ getData, state => state.config, getProjectedData, getDataMin, getDataMax ],
```

Comment:
Just for the sake of consistency maybe we could add another _getter_ for the config to use in the selector, as you've made with `getProjectedData`?
`getConfig = state => state.config || null`
`export const getDomain = createSelector([getData, getConfig, ...], (data, config, ...) = {...})`

Reply:
Ok! Good idea!